doc/source/changelog.rst

```diff
@@ -9,5 +9,26 @@ This document contains the release notes for the project.
 
 .. towncrier release notes start
 
+`0.16.1 <https://github.com/ansys/ansys-sphinx-theme/releases/tag/v0.16.1>`_ - 2024-05-22
+=========================================================================================
+
+### Added
```

Comment:
Hmm.. it seems like some things are not properly adapted @Revathyvenugopal162 - have a look at this:

https://github.com/ansys/pyansys-geometry/blob/26a6e9a6babe1a45a0d783d4c36bfeaac19cb75a/pyproject.toml#L137-L144

and the template:

https://github.com/ansys/pyansys-geometry/blob/main/doc/changelog.d/changelog_template.jinja

Reply:
I believe your main issue is the template in fact. You need to change the underline style

Reply:
Just created #380